com.unity.multiplayer.mlapi/Runtime/Core/NetworkObject.cs

```diff
@@ -313,23 +322,23 @@ public void NetworkHide(ulong clientId)
         /// <param name="clientId">The client to hide the objects from</param>
         public static void NetworkHide(List<NetworkObject> networkObjects, ulong clientId)
         {
-            if (!NetworkManager.Singleton.IsServer)
+            if (networkObjects == null || networkObjects.Count == 0)
             {
-                throw new NotServerException("Only server can change visibility");
+                throw new ArgumentNullException("At least one " + nameof(NetworkObject) + " has to be provided");
```

Comment:
```suggestion
                throw new ArgumentNullException($"At least one {nameof(NetworkObject)} has to be provided");
```

string interpolation is more performant and more modern than string concatenation :)